Makefile

```diff
@@ -18,6 +24,11 @@ setup_linter:
 	go get -u github.com/alecthomas/gometalinter
 	gometalinter --install --update
 
+# Runs dep ensure to populate the vendor directory with
+# the required dependencies and potentially modify Gopkg.lock.
```

Comment:
modifies? :nerd_face:

Reply:
Hm, yes, I worded it a bit weirdly - I meant "To populate ... and potentially modify" 😄